src/_tkagg.cpp

```diff
@@ -296,17 +295,17 @@ int load_tkinter_funcs(void)
             if (!found_tcl) {
                 found_tcl = get_tcl(hMods[i]);
                 if (found_tcl == -1) {
-                    return 1;
+                    return;
```

Comment:
What error is set here instead?

Reply:
good catch... fixed.

Reply:
Actually no, there is indeed an error being set by _dfunc()
```
void *_dfunc(void *lib_handle, const char *func_name)
{
    // Load function `func_name` from `lib_handle`.
    // Set Python exception if we can't find `func_name` in `lib_handle`.
    // Returns function pointer or NULL if not present.

    void* func;
    // Reset errors.
    dlerror();
    func = dlsym(lib_handle, func_name);
    if (func == NULL) {
        PyErr_SetString(PyExc_RuntimeError, dlerror());  // here
    }
    return func;
}
```
(and similarly on Windows).

Left the original version as is.